src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -29,7 +30,7 @@ public Figure getRandomFigure() {
                 int topBase = random.nextInt() * 10;
                 int bottomBase = random.nextInt() * 10;
                 int heightTrapezoid = random.nextInt() * 10;
-                return new IsoscelesTrapezoid(color,topBase, bottomBase, heightTrapezoid);
+                return new IsoscelesTrapezoid(color, topBase, bottomBase, heightTrapezoid);
 
             default:
                 return null;
```

Comment:
you can return default figure instead of null
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/ac1afd0a-a00c-4439-8b52-9682bac0240a)